src/main/java/com/team4/testingsystem/controllers/ChosenOptionController.java

```diff
@@ -0,0 +1,43 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.entities.ChosenOption;
+import com.team4.testingsystem.entities.Test;
+import com.team4.testingsystem.entities.TestQuestionID;
+import com.team4.testingsystem.services.ChosenOptionService;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/chosen_option")
+public class ChosenOptionController {
+
+    private ChosenOptionService chosenOptionService;
+
+    @Autowired
+    public ChosenOptionController(ChosenOptionService chosenOptionService) {
+        this.chosenOptionService = chosenOptionService;
+    }
+
+    @ApiOperation(value = "Get a single chosen option by TestQuestionID")
+    @GetMapping(path = "/")
+    public ChosenOption getById(@RequestBody TestQuestionID testQuestionID) {
+        return chosenOptionService.getById(testQuestionID);
+    }
+
+    @ApiOperation(value = "Use it to get all chosen options from the database")
+    @GetMapping(path = "/options")
+    public Iterable<ChosenOption> getAllByTest(@RequestBody Test test) {
```

Comment:
I think it should be testId as a parameter here. And /{testId} seams to be a good path for it